### DISCUSSION TITLE
How do i stop this from happening

### DISCUSSION THREAD

**Discussion by Camp27** [arkypita/LaserGRBL#1360] [Show and tell]
![20210311_001138](https://user-images.githubusercontent.com/80442906/110739132-94058c80-81fe-11eb-9b2d-00a7a1cdd462.jpg)
When using vectorize, this happens occasionally. How can I stop  it?
![20210311_001143](https://user-images.githubusercontent.com/80442906/110739113-88b26100-81fe-11eb-9647-7f4da2d616a0.jpg)

**Comment by arkypita** [comment]
LaserGRBL.com/faq
